comprehensive/lms/templates/user_dropdown.html

```diff
@@ -55,7 +53,9 @@
             <div role="group" aria-label="User menu" class="user-menu">
                 <button class="dropdown" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span><span class="fa fa-sort-desc" aria-hidden="true"></span></button>
                 <ul class="dropdown-menu" aria-label="More Options" role="menu">
-                    ${navigation_dropdown_menu_links()}
+                    % if request.get_full_path() not in ['/account/link', '/account/link/confirm']:                        
+                        ${navigation_dropdown_menu_links()}
+                    % endif
```

Comment:
are we doing anything to block accessing webpages we don't want the user to access or just removing the hyperlinks? i don't mind either way because the ROI of the former is questionable. Just curious

Reply:
Just removing the links here. We block access through middleware in the platform. We just want to give them the option to sign out here